istio-operator/istio-operator.spec

```diff
@@ -103,13 +107,20 @@ popd
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/istio-operator/%{charts_version}
 pushd OPERATOR/src/github.com/maistra/istio-operator/tmp/_output/
 cp -rpav helm/ $RPM_BUILD_ROOT%{_sysconfdir}/istio-operator/%{charts_version}
+popd
 
+#install manifests
+install -d $RPM_BUILD_ROOT%{_datadir}/%{name}
+cp -ra OPERATOR/src/github.com/maistra/istio-operator/manifests $RPM_BUILD_ROOT%{_datadir}/%{name}
 
 %files
 %{_bindir}/istio-operator
 %{_sysconfdir}/istio-operator
+%{_datadir}/%{name}/manifests
 
 %changelog
+* Thu Jul 25 2019 Dmitri Dolguikh <ddolguik@redhat.com> - 1.0.0-1
```

Comment:
You are not changing the version field, right? So, this should be still 0.12.

Reply:
I rebased my PR on your latest changes, it's correct now :)